Assets/Scripts/BlockBehaviour.cs

```diff
@@ -195,4 +196,14 @@ public void PlayDisplacementSound()
             displacementSound.Play();
         }
     }
+
+    public void SetBoundByGravity()
+    {
+        _boundByGravity = true;
+    }
+
+    public bool IsBoundByGravity()
+    {
+        return _boundByGravity;

```

Comment:
I think we should not have information about gravity for each block. Rather lets change `Moveable` to give us the flexibility to on what faces we can actually interact with. This way we don't need all these getters as well.